src/scene/mesh-instance.js

```diff
@@ -13,6 +13,7 @@ import {
 
 var _tmpAabb = new BoundingBox();
 var _tempBoneAabb = new BoundingBox();
+var _tempSphere = { center: null, radius: 0 };
```

Comment:
I would make this a proper `pc.BoundingSphere` for clarity. If this was Typescript, the compiler would complain. 😄

Reply:
done, makes sense